docs/reference/modules/discovery/zen.asciidoc

```diff
@@ -79,7 +79,8 @@ active master node will step down and a new master election will be begin.
 
 This setting must be set to a quorum of your master eligible nodes. It is recommended to avoid
 having only two master eligible nodes, since a quorum of two is two. Therefore, a loss
-of either master eligible node will result in an inoperable cluster.
+of either master eligible node will result in an inoperable cluster. It is also recommended to keep the 
+number of master nodes below `discovery.zen.minimum_master_nodes * 2` to avoid split brain scenarios.
```

Comment:
We already say in this paragraph that "[t]his setting must be set to a quorum of your master eligible nodes", I think that's enough?